hw/xbox/dsp/dsp.c

```diff
@@ -314,184 +314,3 @@ void dsp_print_registers(DSPState* dsp)
     printf("SP: %02x    SSH: %04x  SSL: %04x\n", 
         dsp->core.registers[DSP_REG_SP], dsp->core.registers[DSP_REG_SSH], dsp->core.registers[DSP_REG_SSL]);
 }
-
```

Comment:
Should we keep the other debugger functions, which are also unused (e.g. `dsp_print_registers`, `dsp_info`, etc.)?

Reply:
No; see my reasoning from PR description:

> These functions weren't suitable to be called from gdb (pointer arguments), and also weren't used by our code.

`dsp_print_registers` and `dsp_info` can be called from gdb and provide valuable information.

By contrast, calling `dsp_get_register_address` from gdb is a lot more complicated and pointless. We can just `print dsp->core.registers[DSP_REG_A0]` instead, or just use `call dsp_print_registers(&dsp)` (which provides the same information).

`dsp_get_register_address` would only be needed if we want to retrieve bits of a single register by name from within our code (like in a GUI, a custom repl, etc.).

Reply:
I'm confused, by this response: did you mean to say *yes* we should keep them, because they can be called from gdb and are useful?

Reply:
- The ones I'm removing are not useful / are hard to call from gdb (so they should be removed).
- The ones I did not remove are useful to be called from gdb (so we should keep them).